.github/workflows/ci.yml

```diff
@@ -8,7 +8,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        laravel: [6, 8, 9]
+        php: ["7.3", "8.0", "8.1"]
+        laravel: ["^6.0", "^8.0", "^9.0", "^10.0"]
```

Comment:
We can drop PHP < 8 support and Laravel < 9 support (across all these PRs). Seems like Laravel < 9 doesn't receive security updates anymore, so we don't need to provide any further support. And L9 requires PHP 8.0, so 7.* isn't needed anymore.

This should simplify the workflow file, the listed|versions in `composer.json`, and make CI runs faster.

Reply:
sweet! give me a couple minutes